menu.rb

```diff
@@ -8,11 +8,11 @@ def greeting
   end
 
   def main_menu
-    puts "\nPlease enter a menu selection: "
-    puts "\n1 - Check your balance"
-    puts "\n2 - Make a withdrawal"
-    puts "\n3 - Make a deposit"
-    puts "\n4 - Exit"
+    puts  "\nPlease enter a menu selection: "
+    puts  "\n1 - Check your balance"
+    puts  "\n2 - Make a withdrawal"
+    puts  "\n3 - Make a deposit"
+    puts  "\n4 - Exit"
     print "\n  => "
     @user_input = gets.chomp
```

Comment:
hint: just because the menu system takes a number as input from the customer does not mean that 'magic' numbers should be passed to the teller.  Passing symbols would make it easier for the teller to decide what to do.  You should be able to get rid of the case statement in teller.